### DISCUSSION TITLE
测试文章

### DISCUSSION THREAD

**Discussion by giscus** [keelii/fresh-blog#1] [Announcements]
# 测试文章

__ you don&

http://localhost:8000/2016/06/10/test-post

**Comment by keelii** [comment]
## 这里可以测试评论功能

Lorem ipsum **dolor** sit *amet*, consectetur adipisicing elit, sed do eiusmodtempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,quis nostrudexercitation ullamco laboris nisi ut aliquip ex ea commodoconsequat. Duis aute irure dolor in reprehenderit in voluptate velit essecillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat nonproident, sunt in culpa qui officia deserunt mollit anim id est laborum.

**Comment by ZhaoTim** [comment]
麻烦问下，博客是开源的吗~

**Reply by keelii** [reply]
博客源代码是基于 deno fresh 实现的，代码可以 clone 过去修改，但是文章内容不开源。
